CHANGELOG.md

```diff
@@ -6,10 +6,12 @@
 * [\#61](https://github.com/Finschia/wasmd/pull/61) bumpup ibc-go to v4
 
 ### Improvements
+* [\#64](https://github.com/Finschia/wasmd/pull/64) test: add test cases to confirm output for PinnedCodes
 * [\#63](https://github.com/Finschia/wasmd/pull/63) add event checking to TestStoreCode
 * [\#65](https://github.com/Finschia/wasmd/pull/65) add test cases for empty request in each function
 * [\#66](https://github.com/Finschia/wasmd/pull/66) add test cases for invalid pagination key in some functions
 
+
```

Comment:
```suggestion
```

Reply:
This blank line before `###` is needed, isn't it?

Reply:
Yes, but it seems you have two blank lines before.